### ISSUE TITLE
rename and migrate `ChatResponsePending` accessibility signal to generic progress one

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210590]
Now that notebooks and other components that have progress use this accessibility signal, it should be named something generic and migrated.
